jaeger-core/src/main/java/com/uber/jaeger/reporters/protocols/ThriftSpanConverter.java

```diff
@@ -74,7 +74,7 @@
     List<LogData> logs = span.getLogs();
     if (logs != null) {
       for (LogData logData : logs) {
-        annotations.add(new Annotation(logData.getTime(), logData.getMessage()));
+        annotations.add(new Annotation(logData.getTime(), logData.getMessage()).setHost(host));
```

Comment:
This might be something that only the zipkin elasticsearch span store is running into. I'm going to try and verify against other storages. I totally agree that it'd bloat span sizes which is why I was a little hesitant to add it.